client/my-sites/plugins/education-footer/index.tsx

```diff
@@ -0,0 +1,101 @@
+import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
+import FooterSection from 'calypso/components/footer-section';
+import LinkCard from 'calypso/components/link-card'; // should we host this one inside footer-section?
+
+interface Article {
+	title: string;
+	category: string;
+	url: string;
+	background: string;
+}
+
+const deslugify = ( string: string ) => string.replaceAll( '-', ' ' );
+const slugify = ( string: string ) =>
+	string
+		.replace( /[&/\\#,+()$~%.'":*?<>{}’]/g, '' )
+		.toLowerCase()
+		.replaceAll( ' ', '-' );
+
+const ArticleLinksContainer = styled.div`
+	@media ( max-width: 960px ) {
+		grid-template-columns: repeat( 1, 1fr );
+	}
+
+	@media ( max-width: 660px ) {
+		padding: 0 16px;
+	}
+
+	display: grid;
+	grid-template-columns: repeat( 3, 1fr );
+	column-gap: 29px;
+	row-gap: 10px;
+`;
+
+const EducationFooter = () => {
+	const translate = useTranslate();
+
+	const articles: Article[] = [
+		{
+			category: translate( 'website-building', {
+				comment:
+					'Wordpress.com Go article (https://wordpress.com/go/) category on kebab case, example: Website Building = website-bulding',
+				textOnly: true,
+			} ),
+			title: translate( 'What Are WordPress Plugins and Themes? (A Beginner’s Guide)', {
+				comment: 'Wordpress.com Go article title (https://wordpress.com/go/)',
+				textOnly: true,
+			} ),
+			url:
+				'https://wordpress.com/go/website-building/what-are-wordpress-plugins-and-themes-a-beginners-guide/',
```

Comment:
These can be localized too:

```suggestion
		// earlier
		import { useLocalizeUrl } from '@automattic/i18n-utils';
		const localizeUrl = useLocalizeUrl();

		// here
		url: localizeUrl( 'https://wordpress.com/go/website-building/what-are-wordpress-plugins-and-themes-a-beginners-guide/' ),
```

Reply:
Oh good catch, didn't know we could localize urls!

Reply:
✅

Reply:
Yea, basically anything on wordpress.com/* is supposed to be supported, so all support sites, /go/ and marketing pages should be mapped.